cypress/e2e/3-testing/login-logout.cy.js

```diff
@@ -0,0 +1,64 @@
+const user = {
+  email: "lhvk@stud.noroff.no",
```

Comment:
A good thing to put in .env

Reply:
After trying several methods that did not work, I settled with a cypress.env.json file where I stored my sensitive data. This file is untracked, but I added an example file for others to use. At least the email and password is now dynamic for whoever want to test it out. I hope this is sufficient. Thanks